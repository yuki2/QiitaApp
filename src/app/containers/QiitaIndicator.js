import React from 'react';
import { ActivityIndicator, StyleSheet } from 'react-native';

const styles = StyleSheet.create({
  loading: {
    flex: 1,
    alignItems: 'center',
    justifyContent: 'center',
    backgroundColor: 'white',
  },
});

const QiitaIndicator = () => <ActivityIndicator animating style={styles.loading} size="large" />;

export default QiitaIndicator;
