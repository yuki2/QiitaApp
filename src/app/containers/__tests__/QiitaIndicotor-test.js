import React from 'react';
import renderer from 'react-test-renderer';
import QiitaIndicator from '../QiitaIndicator';

it('renders correctly', () => {
  const tree = renderer.create(<QiitaIndicator />).toJSON();
  expect(tree).toMatchSnapshot();
});
