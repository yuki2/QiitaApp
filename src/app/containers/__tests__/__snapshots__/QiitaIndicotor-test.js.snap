// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ActivityIndicator
  animating={true}
  color="#999999"
  hidesWhenStopped={true}
  size="large"
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "center",
    }
  }
/>
`;
